third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -44,16 +42,36 @@ class CallOpConversion : public mlir::RewritePattern {
            llvm::StringRef::npos;
   }
 
-  bool isPredicatedLoadNT(LLVM::CallOp callOp) const {
+  bool isPredicatedLoadCA(LLVM::CallOp callOp) const {
     return callOp.getCallee().value().find(
-               mlir::LLVM::AMD::Predicated_Load_NT) != llvm::StringRef::npos;
+               mlir::LLVM::AMD::Predicated_Load_CA) != llvm::StringRef::npos;
```

Comment:
Note that `StringRef::contains` is a better API to do the check here. Similarly for the following calls.